modules/rosa-create-an-identity-based-policy.adoc

```diff
@@ -149,11 +54,7 @@ You can create an identity-based Identity and Access Management (IAM) policy tha
 }
 ----
 +
-[NOTE]
-====
-This list is subject to change. Additionally, you must specify the IP addresses in CIDR notation.
-====
-+
+. Copy and paste the `policy_document.json` with all the latest IP addresses into the editor.
```

Comment:
The phrasing with "latest" seems potentially problematic from a localization perspective.

```suggestion
. Copy and paste the `policy_document.json` file contents, including the updated IP addresses, into the editor.
```

Reply:
Made the update.

Reply:
Please see my comment: https://github.com/openshift/openshift-docs/pull/70714#discussion_r1475702026